layouts/default/alice/info.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
If you enable "fast input" mode on the configurator, the order of keys is not correct.